lib/utils/FileCache.js

```diff
@@ -28,6 +28,13 @@ class FileCache {
 		this._hashOfConfig = hashOfConfig;
 	}
```

Comment:
[suggest] Is it possible to simplify the `FileCache` constructor like this?

```js
class FileCache {
	/**
	 * @param {string | undefined} cacheLocation
	 * @param {string} cwd
	 */
	constructor(cacheLocation, cwd) {
```

Reply:
I removed `hashOfConfig` from argments of `constructor`.  [`ca2fb53`](https://github.com/stylelint/stylelint/pull/6356/commits/ca2fb534ba93e2dba40092945e6f56758c99a97e#diff-83240954c077d79828b7a92f75b9e1c412d2b228b367c5864011f57b937fd26e)

Reply:
Thanks. In addition, can you please fix also the JSDoc like this?

```js

class FileCache {
	/**
	 * @param {string | undefined} cacheLocation
	 * @param {string} cwd
	 */
	constructor(cacheLocation = DEFAULT_CACHE_LOCATION, cwd) {
```

- Move the JSDoc from `class FileCache` to `constructor`
- Fix the `cacheLocation` parameter type
- Add the `cwd` parameter and make it required (removing `= process.cwd()`)

Reply:
Updated JSDoc of `FileCache`. [`b9c8a00`](https://github.com/stylelint/stylelint/pull/6356/commits/b9c8a00f7cf4abf58dafd3b0d0660703aa6c913d#diff-83240954c077d79828b7a92f75b9e1c412d2b228b367c5864011f57b937fd26e)